package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kentico/kontent-cli",
-  "version": "0.3.0",
+  "version": "0.5.0",
```

Comment:
Why raise by 2? I guess because 2 features were added, but I think we can still have just 0.4.0 with 2 features.

Reply:
it's just because I accidentally released 0.4.0-next-1 instead of 0.3.0-next-1 during testing